app/models/organization.rb

```diff
@@ -56,13 +56,39 @@ def one_owner_remains?
     users.count == 1
   end
 
+  def organization_webhook_active?
+    return false if webhook_id.nil?
+
+    begin
+      github_client.org_hook(github_id, webhook_id).active
+    rescue Octokit::NotFound
```

Comment:
this could also raise `Octokit::Forbidden` but we should be checking that the token has enough scopes before making the call.